Sources/MountebankSwift/Api/Mountebank+Completion.swift

```diff
@@ -0,0 +1,305 @@
+import Foundation
+
+extension Mountebank {
+
+    // MARK: - Imposter
+
+    /// Get a single Imposter
+    ///
+    /// Retrieving an ``Imposter`` is generally useful for one the following reasons:
+    /// - You want to perform mock verifications by inspecting the requests array).
+    /// - You want to debug stub predicates by inspecting the matches array (you must run mb with the --debug
+    ///   command line parameter for this to work).
+    /// - You proxied a real service, and want to be save off the saved responses in a subsequent disconnected test run.
+    ///
+    /// - Parameters:
+    ///   - port: The ``Imposter`` server port
+    ///   - replayable: Set to `true` to retrieve the minimum amount of information for creating the imposter in the
+    ///     future. This leaves out the requests array and any hypermedia.
+    ///   - removeProxies: Set to `true` to remove all proxy responses (and ``Stubs``) from the response. This is useful
+    ///     in record-playback scenarios where you want to seed the imposters with proxy information but leave it out on
+    ///     subsequent test runs. You can recreate the ``Imposter`` in the future by using the response.
+    ///   - completion: Completion with a ``Imposter`` that listens to the provided port
```

Comment:
It would be so much nicer if you could just say that these docs should be generated based on the docs for each properties. Now we have to maintain the same set 3 times.. 

But great that you've added them

Reply:
Is this possible? Also the docs are a bit different.

Reply:
No I don't that would be possible :p